api/admin.py

```diff
@@ -0,0 +1,47 @@
+# This file is a part of GreedyBear https://github.com/honeynet/GreedyBear
+# See the file 'LICENSE' for copying permission.
+
+
+from django.contrib import admin
+from durin.admin import AuthTokenAdmin
+from durin.models import AuthToken, Client
+
+
+# Auth Token stuff
+class CustomAuthTokenAdmin(AuthTokenAdmin):
+    """
+    Custom admin view for AuthToken model
+    """
+
+    exclude = []
+    raw_id_fields = ("user",)
+    readonly_fields = ("token", "expiry", "created", "expires_in")
+
+    def get_fieldsets(self, request, obj=None):
+        if not obj:
+            return [
+                (
+                    "Create token for Greedybear",
```

Comment:
also this is copy-pasted from IntelOwl but this is a customization made for the PyIntelOwl client and, right now, there is no GreedyBear client.

Even if we could still add something similar for an eventual future PyGreedyBear:
* you badly copy-pasted...you removed `admin.site.unregister(Client)`
* you missed a part of the copy-paste, a critical migration where the clients are created

Reply:
Yes , I am not clear about the code here(the methods used in the class), all the know is to register the authtoken , so that they can be view them via admin portal. 
Its my mistake, I should be asking u quesn regarding this rather than doing this.
Can u just give an idea of what changes I have to make?

Reply:
"Use how these packages are integrated in IntelOwl for reference" (see #26) does not mean that you just copy-paste without even knowing what you are doing. These kind of contributions:
* waste time of maintainers
* waste time of contributors
* they bring NO value at all.

We leave these issues open because we want to find new possible contributors and this is a way to evaluate them. Otherwise, we would have just done this task on our own very fast.

We don't want people who just go at the end of the goal as fast as possible without even understanding what they are doing.